.github/workflows/publish-demo.yml

```diff
@@ -55,18 +55,19 @@ jobs:
       - name: Set up Docker Buildx
         uses: docker/setup-buildx-action@v2
 
-      - name: Login to DockerHub
+      - name: Login to Registry
         uses: docker/login-action@v3
         with:
-          username: ${{ secrets.DOCKERHUB_USERNAME }}
-          password: ${{ secrets.DOCKERHUB_KEY }}
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Build and push image
         uses: docker/build-push-action@v4
         with:
           context: ./mobidam-sst-management-backend
           push: true
-          tags: itatm/mobidam:mobidam-sst-management-backend-demo
+          tags: ghcr.io/it-at-m/mobidam-sst-management:backend-demo
```

Comment:
ghcr.io/it-at-m/mobidam-sst-management-backend:demo
Sonst haben wir doch wieder backend und frontend in einem.